ee/server/calendar/google_calendar.go

```diff
@@ -192,35 +191,41 @@ func (lowLevelAPI *GoogleCalendarLowLevelAPI) DeleteEvent(id string) error {
 }
 
 func (lowLevelAPI *GoogleCalendarLowLevelAPI) Watch(eventUUID string, channelID string, ttl uint64) (resourceID string, err error) {
-	resp, err := lowLevelAPI.withRetry(
-		func() (any, error) {
-			return lowLevelAPI.service.Events.Watch(calendarID, &calendar.Channel{
-				Id:   channelID, // channelID is also used for authentication -- it should be a random value
-				Type: "web_hook",
-				Address: fmt.Sprintf("%s/api/v1/fleet/calendar/webhook/%s",
-					lowLevelAPI.serverURL, eventUUID),
-				Params: map[string]string{
-					"ttl": strconv.FormatUint(ttl, 10),
-				},
-			}).EventTypes("default").Do()
-		},
-	)
-	if err != nil {
-		return "", err
-	}
-	return resp.(*calendar.Channel).ResourceId, nil
+	// Disabling this feature to address bugs
+	return "", nil
```

Comment:
All events will have empty `resource_id` in the details, any issues with this?

Reply:
looking upstream it doesn't seem to be used anywhere